lib/trento/infrastructure/discovery/payloads/cloud_discovery_payload.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Trento.Integration.Discovery.CloudDiscoveryPayload do
+defmodule Trento.Infrastructure.Discovery.CloudDiscoveryPayload do
```

Comment:
`Infrastructure.Discovery.Payloads.CloudDiscoveryPayload`
The same for the other payloads

Reply:
done